index.html

```diff
@@ -54,7 +54,7 @@ <h2>2022.08.27(THU)~30(FRI)</h2>
             <small>@Halloway Center Sydney, Australia</small>
         </div>
     </section>
-    <section class="program-dets">
+    <section id="program-dets">
         <h3>Main Program</h3>
         <div class="grid">
         <div class="p-grid">
```

Comment:
Please make sure all of your articles for both pages use their corresponding semantic tag.

<img width="779" alt="Screen Shot 2022-07-27 at 9 27 40" src="https://user-images.githubusercontent.com/82426682/181272960-d335041b-4437-451a-b71c-05524de002ae.png">